Server/Network/MessageProcessor.cs

```diff
@@ -231,7 +231,7 @@ internal static void ProcessData(Client client, string[] parse)
                                 }
                                 else
                                 {
-                                    Messenger.PlainMsg(client, "Character does not exist", Enums.PlainMsgType.Chars);
+                                    Messenger.PlainMsg(client, "This character hasn't been sent in yet!", Enums.PlainMsgType.Chars);
```

Comment:
Can this line be more descriptive? I think new players will be confused by what this actually means